Dockerfile

```diff
@@ -16,11 +16,10 @@
 #
 FROM alpine:latest as pre-build
 
-ARG APISIX_DASHBOARD_VERSION=master
+COPY . /usr/local/apisix-dashboard
```

Comment:
The "./docs/en/latest/deploy-with-docker.md"  is using this environment, don't forget to modify this.

Reply:
OK, I deleted them